engine/extensions-pentaho-metadata/src/main/java/org/pentaho/reporting/engine/classic/extensions/datasources/pmd/SimplePmdDataFactory.java

```diff
@@ -92,6 +94,7 @@ protected boolean isExpandArrays() {
   private String domainId;
   private String xmiFile;
   private IPmdConnectionProvider connectionProvider;
+  private boolean useParentIdentifier;
```

Comment:
Do you think that declaring this boolean property default value would improve readability? 
```suggestion
  private boolean useParentIdentifier = false;
```

Reply:
Yeah, perhaps. I don't see any problem, sounds good 👍

Reply:
Perhaps. Ideally I would like to setup the variable with the value from config straight away, but I have to do it in the `initialize` function as I need to have the configurations loaded, therefore I cannot do it in the constructor (in a static block, for example). Either way, the 2 declarations above are equivalent, so I don't see any reason to oppose your suggestion.

Reply:
@befc made the changes 👍